package.json

```diff
@@ -45,7 +47,10 @@
   },
   "main": "lib/protractor.js",
   "scripts": {
-    "pretest": "./scripts/pretest.sh",
+    "prepublish": "./node_modules/.bin/typings install && ./node_modules/typescript/bin/tsc && cp built/*.js lib/",
```

Comment:
I think I was wrong about this. You have typings in devDependencies, so the typings install command belongs in postinstall script (otherwise your teammates won't be able to compile until they attempt to publish!)